Orange/widgets/data/oweditdomain.py

```diff
@@ -662,6 +673,168 @@ def on_label_selection_changed(self):
+    ) -> None:
+        super().__init__(parent, flags, **kwargs)
+        self.variable = variable
+        self.data = data
+        self.selected_attributes = selected_attributes
+
+        # grouping strategy
+        self.selected_radio = radio1 = QRadioButton("Group selected values")
+        self.frequent_abs_radio = radio2 = QRadioButton(
+            "Group values with less than"
+        )
+        self.frequent_rel_radio = radio3 = QRadioButton(
+            "Group values with less than"
+        )
+        self.n_values_radio = radio4 = QRadioButton(
+            "Group all except"
+        )
+
+        # if selected attributes available check the first radio button,
+        # it is most probable option
+        if selected_attributes:
+            radio1.setChecked(True)
+        else:
+            radio2.setChecked(True)
+
+        label2 = QLabel("occurrences")
+        label3 = QLabel("occurrences")
+        label4 = QLabel("the most frequent values")
+
+        self.frequent_abs_spin = spin2 = QSpinBox()
+        max_val = len(data)
+        spin2.setMinimum(1)
+        spin2.setMaximum(max_val)
+        spin2.setValue(10)
+        spin2.setMinimumWidth(
+            self.fontMetrics().width("X") * (len(str(max_val)) + 1) + 20
+        )
+        spin2.valueChanged.connect(partial(self._spin_changed, spin2))
+
+        self.frequent_rel_spin = spin3 = QDoubleSpinBox()
+        spin3.setMinimum(0)
+        spin3.setDecimals(1)
+        spin3.setSingleStep(0.1)
+        spin3.setMaximum(100)
+        spin3.setValue(10)
+        spin3.setMinimumWidth(self.fontMetrics().width("X") * (2 + 1) + 20)
+        spin3.setSuffix(" %")
+        spin3.valueChanged.connect(partial(self._spin_changed, spin3))
+
+        self.n_values_spin = spin4 = QSpinBox()
+        max_val = min(10, len(variable.categories))
+        spin4.setMinimum(0)
+        spin4.setMaximum(len(variable.categories))
+        spin4.setValue(max_val)
+        spin4.setMinimumWidth(
+            self.fontMetrics().width("X") * (len(str(max_val)) + 1) + 20
+        )
+        spin4.valueChanged.connect(partial(self._spin_changed, spin4))
+
+        grid_layout = QGridLayout()
+        # first row
+        grid_layout.addWidget(radio1, 0, 0, 1, 2)
+        # second row
+        grid_layout.addWidget(radio2, 1, 0, 1, 2)
+        grid_layout.addWidget(spin2, 1, 2)
+        grid_layout.addWidget(label2, 1, 3)
+        # third row
+        grid_layout.addWidget(radio3, 2, 0, 1, 2)
+        grid_layout.addWidget(spin3, 2, 2)
+        grid_layout.addWidget(label3, 2, 3)
+        # fourth row
+        grid_layout.addWidget(radio4, 3, 0)
+        grid_layout.addWidget(spin4, 3, 1)
+        grid_layout.addWidget(label4, 3, 2, 1, 2)
+
+        group_box = QGroupBox("Grouping strategy")
+        group_box.setLayout(grid_layout)
+
+        # grouped variable name
+        new_name_label = QLabel("New value name: ")
+        self.new_name_line_edit = n_line_edit = QLineEdit("Others")
```

Comment:
I'm not sure, but think it would be better to call this "Other"? Like in zoo, type can be "mammal", "fish", "insect" or "other". Or, for instance, "cause" would be "human error", "mechanical error", "other".